test/functional/eval/reg_spec.lua

```diff
@@ -0,0 +1,16 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
let's put all of the test in macro_spec.lua. there is no reason for them to be separate really.